packages/backend-core/src/security/permissions.ts

```diff
@@ -78,7 +78,6 @@ export const BUILTIN_PERMISSIONS = {
     permissions: [
       new Permission(PermissionType.QUERY, PermissionLevel.READ),
       new Permission(PermissionType.TABLE, PermissionLevel.READ),
-      new Permission(PermissionType.VIEW, PermissionLevel.READ),
```

Comment:
How are we giving these permissions to the users? Can this break the current views somehow?

Reply:
we don't need the view permission anymore, it has been swapped to table permission as that is closer based on how the row API is used.